frontend/public/components/_resource-dropdown.scss

```diff
@@ -55,6 +55,7 @@
 
 .co-search-group__resource {
   margin: 0 15px 15px 0;
+  width: auto !important;
```

Comment:
This is less than ideal, but there wasn't another way to overwrite the patternfly width on just the events page.  `.co-search-group__resource` is only used on that page.